packages/documentation-v7/package.json

```diff
@@ -31,10 +31,12 @@
   },
   "devDependencies": {
     "@geometricpanda/storybook-addon-badges": "1.1.1",
+    "@storybook/addon-a11y": "6.5.16",
     "@storybook/addon-essentials": "7.0.26",
     "@storybook/addon-links": "7.0.26",
     "@storybook/blocks": "7.0.26",
     "@storybook/manager-api": "^7.0.26",
+    "@storybook/testing-library": "0.0.14-next.2",
```

Comment:
```suggestion
```
It does not make any sense to install it, but not to use it in the main.ts file.
And this addon only works together with some other modules, which are not installed yet:
https://storybook.js.org/docs/web-components/writing-tests/interaction-testing